app/views/disciplines/show.erb

```diff
@@ -5,6 +5,9 @@
     <%= @discipline.description %>
   </div>
 </div>
-<% if current_user && current_user.admin? %>
-  <%= link_to "edit discipline", edit_discipline_path(@discipline), class: "btn btn-xs btn-primary" %>
-<% end %>
+<div class="row">
+  <% if current_user&.admin? %>
+    <%= link_to "Edit discipline", edit_discipline_path(@discipline), class: "btn btn-xs btn-primary" %>
+    <%= link_to "Delete discipline", discipline_path(@discipline), method: :delete, data: {confirm: "Are you sure?"}, class: "btn btn-xs btn-danger" %>
```

Comment:
no spaces in the hash, and you're using an english warning here while the rest of the app is in polish. Please stay consistent :)